sklearn/preprocessing/_label.py

```diff
@@ -56,6 +56,10 @@ def _encode_numpy(values, uniques=None, encode=False, check_unknown=True):
 
 def _encode_python(values, uniques=None, encode=False):
     # only used in _encode below, see docstring there for details
+    if None in values:
```

Comment:
This is no longer necessary. I'd rather avoid it as it is computational expense for a rare case.

Reply:
It were there because @ogrisel suggested a more detailed information, but I'll delete those lines then.